openwisp_notifications/static/openwisp-notifications/js/notifications.js

```diff
@@ -11,9 +12,32 @@ if (typeof gettext === 'undefined') {
         notificationWidget($);
         initNotificationDropDown($);
         initWebSockets($);
+        owNotificationWindow.set();
     });
+    $(window).on('beforeunload', function () {owNotificationWindow.remove();});
+    $(window).on('storage', function () {owNotificationWindow.set();});
 })(django.jQuery);
 
+const owNotificationWindow = {
+    set: function(){
+        // Sets the first opened window
+        // Sets another window when first opened window get closed
+        if (localStorage.getItem('owWindow') === null) {
```

Comment:
Doing this check here will break the implementation. 

When a user will change focus to another window, value of `localStorage.getItem('owWindow')` will not be `null`, hence the the notification alert won't be played from the right window. 

Why did you make this change?

Reply:
I though playing of sound is more important. Also when in previous code when last opened window gets closed then also sound is not played from the focused window. 

If you think last opened window should play sound then we can change it. But user can traverse to any window at any time that's why I didn't consider it too change every time when a new window is opened.